src/Extensions.Base.Api.Tests/Extensions.Base.Api.Tests.csproj

```diff
@@ -2,8 +2,10 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved.
      Licensed under the MIT License. -->
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), EnlistmentRoot.marker))\build\Common.props" />
-  <Import Project="$(EnlistmentRoot)\build\Managed.props" />
+  <PropertyGroup>
+    <UseDotNetSdk>false</UseDotNetSdk>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), EnlistmentRoot.marker))\build\Managed.props" />
```

Comment:
>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), EnlistmentRoot.marker)) [](start = 19, length = 93)

$(EnlistmentRoot) ?

Reply:
This is the first import of the project so the EnlistmentRoot property hasn't been set yet.